src/helpers/randomId.js

```diff
@@ -0,0 +1,3 @@
+const randomIdGenerator = () => Math.floor((Math.random() * 100));
```

Comment:
**[OPTIONAL]**
This is definitely not an efficient way to generate unique ids, random numbers are not necessarily unique, but random numbers generated in a non-consistent manner. I suggest the use of packages like `short-id` or `UUID`

Reply:
Done :heavy_check_mark: